features/support/wiremock.js

```diff
@@ -6,25 +6,25 @@ let externalActivityPub;
 /** @type {WireMock} */
 let ghostActivityPub;
 
-export function getExternalActivityPub() {
+export function getExternalWiremock() {
     if (!externalActivityPub) {
-        externalActivityPub = new WireMock(
-            process.env.URL_EXTERNAL_ACTIVITY_PUB,
-        );
+        externalActivityPub = new WireMock('http://fake-external-activitypub');
```

Comment:
The wiremock hostnames now differ from the URLs because they are proxied to via Caddy - I've hardcoded for now